skipper/cli.py

```diff
@@ -25,8 +25,9 @@ def cli(ctx, registry, build_container_image, build_container_tag, build_contain
 
     ctx.obj['registry'] = registry
     ctx.obj['build_container_image'] = build_container_image
-    ctx.obj['build_container_tag'] = build_container_tag
     ctx.obj['build_container_net'] = build_container_net
+    ctx.obj['git_revision'] = build_container_tag == 'git:revision'
```

Comment:
why do you need to propagate this value to commands functions? 
you can simply set `build_container_tag` with the current git revision, if `build_container_tag == git:revision`:
```python 
ctx.obj['build_container_tag'] = git.get_hash() if build_container_tag == 'git:revision' else build_container_tag
```